airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -142,3 +143,37 @@ def patch_dag_run(
     dag_run = session.get(DagRun, dag_run.id)
 
     return DAGRunResponse.model_validate(dag_run, from_attributes=True)
+
+
+@dag_run_router.get(
+    "/{dag_run_id}/upstreamAssetEvents",
+    responses=create_openapi_http_exception_doc(
+        [
+            status.HTTP_401_UNAUTHORIZED,
+            status.HTTP_403_FORBIDDEN,
```

Comment:
We need to remove that. (same as https://github.com/apache/airflow/pull/43934#discussion_r1842264554)

Reply:
Oh ok, wasnt aware of this one. Making the change